app/monitoringapi.go

```diff
@@ -186,17 +186,27 @@ func beaconNodeSyncing(ctx context.Context, eth2Cl eth2client.NodeSyncingProvide
 	return state.IsSyncing, nil
 }
 
-// peerCounter populates the peerCountGauge with the beacon node peer count.
-func peerCounter(ctx context.Context, eth2Cl eth2wrap.Client, clock clockwork.Clock) {
-	ticker := clock.NewTicker(1 * time.Minute)
-
+// beaconNodeMetrics sets beacon node metrics like the peer count and node version.
+func beaconNodeMetrics(ctx context.Context, eth2Cl eth2wrap.Client, clock clockwork.Clock) {
+	peerCountTicker := clock.NewTicker(1 * time.Minute)
 	setPeerCount := func() {
-		count, err := beaconNodePeerCount(ctx, eth2Cl)
+		peerCount, err := eth2Cl.NodePeerCount(ctx)
 		if err != nil {
 			log.Error(ctx, "Failed to get beacon node peer count", err)
 			return
 		}
-		peerCountGauge.Set(float64(count))
+		beaconNodePeerCountGauge.Set(float64(peerCount))
+	}
+
+	nodeVersionTicker := clock.NewTicker(10 * time.Minute)
+	setNodeVersion := func() {
+		version, err := eth2Cl.NodeVersion(ctx)
+		if err != nil {
+			log.Error(ctx, "Failed to get beacon node version", err)
+			return
+		}
+
+		beaconNodeVersionGauge.WithLabelValues(version).Set(1)
```

Comment:
see the warning I added to the issue: https://github.com/ObolNetwork/charon/issues/1773#issuecomment-1423718659